### PR TITLE
*: random cleanups

### PR DESCRIPTION
**Pull request by suzaku**
### What problem does this PR solve? 

Some random cleanups done while read related code.

### What is changed and how it works?

1. Change some variable names to improve readability
1. Add a missing error check
1. Reduce nested level of some code

### Check List 

Tests 

 - Unit test

### CODE REVIEW

File: cdc/kv/client.go [hunk 1]
```diff
@@ -361,24 +360,24 @@ func (c *CDCClient) divideAndSendEventFeedToRegions(
 func (c *CDCClient) singleEventFeed(
 	ctx context.Context,
 	span util.Span,
-	ts uint64,
-	meta *metapb.Region,
+	checkpoint uint64,
```
Comment (leoppro) [inline cdc/kv/client.go]: hoe about `checkPointTs`?

Comment (suzaku) [reply cdc/kv/client.go]: Yes. Note that checkpoint is a valid word, not two words, so we should use `checkpoint` instead of `checkPoint`.

**Pull request merged**